docs/install-upgrade/upgrade.md

```diff
@@ -5,30 +5,33 @@
 Starting with Beta-1 release and onwards, the upgrade process is more streamlined and fully automated. The control node
 is upgraded in place and the agents/switches is upgraded using the control node.
 
-In order to apply the upgrade, the following steps need to be followed:
+In order to apply the upgrade, use the following instructions:
 
-- use the `hhfab` directory from the initial deployment or init the new one using the configs from the running installation
-- run `hhfab build --mode=manual` to generate fully self-contained (airgap) upgrade package
-  - for control node named `control-1` it will be `result/control-1-install.tgz`
-- upload it to the control node (e.g. using `scp`)
-- unpack and run `hhfab-recipe control upgrade` from the resulting directory
+1. use the `hhfab` directory from the initial deployment or init the new one
```

Comment:
I think we should emphasize that you `fab.yaml` should be up-to-date with the actual config to avoid breaking things. Maybe even suggest to always init from the config just exported from API?

Reply:
Done